src/xcode/ENA/ENA/Source/Scenes/ContactDiary/Model/DiaryExportItem.swift

```diff
@@ -0,0 +1,42 @@
+////
+// 🦠 Corona-Warn-App
+//
+
+import UIKit
+
+class DiaryExportItem: NSObject, UIActivityItemSource {
+
+	// MARK: - Init
+	
+	init(subject: String, body: String) {
+		self.subject = subject
+		self.body = body
+	}
+	
+	// MARK: - Protocol UIActivityItemSource
+	
+	func activityViewControllerPlaceholderItem(_ activityViewController: UIActivityViewController) -> Any {
+		return ""
+	}
+	
+	func activityViewController(_ activityViewController: UIActivityViewController, itemForActivityType activityType: UIActivity.ActivityType?) -> Any? {
+		return body
+	}
+	
+	func activityViewController(_ activityViewController: UIActivityViewController, subjectForActivityType activityType: UIActivity.ActivityType?) -> String {
+		guard let activityType = activityType else {
+			return ""
+		}
+		switch activityType {
+		case .mail:
+			return subject
```

Comment:
Just an idea to make it shorter:

```suggestion
		switch activityType {
		case .mail?:
			return subject
```